src/quo/components/profile/expanded_collectible/view.cljs

```diff
@@ -55,7 +55,7 @@
                      {:label   (i18n/label :t/cant-fetch-info)
                       :counter counter
                       :theme   theme}]
-       [rn/view {:style {:flex 1}}
```

Comment:
hmm, are you sure about this? The component looked right to me in the preview screen. 
Can you explain why you did this?

Reply:
@J-Son89, there is no reason for `expanded-collectible` to take all available space, space of the images should be enough.
I ran into a problem when placed components on the screen. Components order:
```
[quo/expanded-collectible]
[quo/network-tags]
[quo/amount-input]
...
```
Since expanded-collectible tried to occupy as much space as possible, it was pushing away `network-tags` making it impossible to maintain correct margin between `expanded-collectible` and `network-tags`

Reply:
Thanks @vjkr, the thing is if the image is portrait it should respect that aspect ratio and the same for landscape. As long as that's still maintained then sounds good.

Thanks for the very clear explanation! 🙏

Reply:
Perfect, thanks!